flake/dev/generate-all-maintainers/extract-maintainers-meta.nix

```diff
@@ -0,0 +1,57 @@
+# Extract maintainers from nixvim configuration using meta.maintainers
+# This script evaluates an empty nixvimConfiguration and extracts the merged maintainer information
+let
+  nixvim = import ../../..;
+  pkgs = import <nixpkgs> { };
```

Comment:
If we're importing nixvim above (using flake-compat) we can access `nixvim.inputs.nixpkgs`.

Either way, if we don't need an actual pkgs instance, we can either import `<nixpkgs/lib>` or use `nixvim.inputs.nixpkgs.lib` to get `lib` without evaluating pkgs.

Alternatively, we can get nixvim's extended lib from its empty configuration, since we'll be evaluating that anyway:

```
inherit (nixvim.nixvimConfigurations.${builtins.currentSystem}.default._module.specialArgs) lib;
```